Flow.Launcher.Core/ExternalPlugins/Environments/AbstractPluginEnvironment.cs

```diff
@@ -37,10 +38,10 @@ internal AbstractPluginEnvironment(List<PluginMetadata> pluginMetadataList, Plug
             PluginSettings = pluginSettings;
         }
 
-        internal IEnumerable<PluginPair> Setup()
```

Comment:
why would you not use a generic version of task?

Reply:
I have fixed this.